adhocracy-plus/templates/a4modules/module_detail.html

```diff
@@ -59,22 +48,6 @@
             <div class="container">
                 <div class="row mb-5">
                     <div class="col-md-8 col-lg-6 offset-md-2 offset-lg-3">
-                        {% with container=project.containers.first %}
-                            {% if container %}
-                                <a href="{{ container.get_absolute_url }}" class="project-header__link" data-embed-target="external">
-                                    <i class="fa fa-arrow-left" aria-hidden="true"></i>
-                                    {% blocktrans with name=container.name %}to project collection {{ name }}{% endblocktrans %}
```

Comment:
Do we want to remove that for all modules?

Reply:
I didn't find any trace of project containers being used as it seemed like a left over from mB so thought I would clean it up

Reply:
Ah right